scripts/install.sh

```
@@ -0,0 +1,282 @@
+    fi
+    
+    if [ -z "$VERSION" ]; then
+        error "Failed to get latest version"
+    fi
+    
+    log "Latest version: $VERSION"
+}
+
+# Download and install binaries
+install_binaries() {
+    local tmpdir=$(mktemp -d)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)